install/cockroach.go

```diff
@@ -69,7 +70,7 @@ func getCockroachVersion(c *SyncedCluster, i int, host, user string) (*version.V
 	cmd := cockroachNodeBinary(c, i) + " version"
 	out, err := session.CombinedOutput(cmd)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrapf(err, "%s", out)
```

Comment:
Can't these be replaced with `errors.Wrap(err, out)`? If so, slight preference everywhere to create strings with `errors.Wrap(err, out).Error()`.

Reply:
`out` is a `[]byte`. The default formatting for `[]byte` is unreadable. I could make this `errors.Wrap(err, string(out))`, though I prefer the explicitness of `errors.Wrapf`.

Reply:
In that case, I like this approach better too.